propagator/propagator.go

```diff
@@ -38,9 +38,13 @@ const traceContextHeaderFormat = "^(?P<trace_id>[0-9a-f]{32})/(?P<span_id>[0-9]{
 // traceContextHeaderRe is a regular expression object of TraceContextHeaderFormat.
 var traceContextHeaderRe = regexp.MustCompile(traceContextHeaderFormat)
 
-// traceContextHeaders is the list of headers that are propagated. Cloud Trace only requires
+// cloudTraceContextHeaders is the list of headers that are propagated. Cloud Trace only requires
 // one element in the list.
-var traceContextHeaders = []string{TraceContextHeaderName}
+var cloudTraceContextHeaders = []string{TraceContextHeaderName}
+
+// TraceparentHeaderName is the HTTP header field for w3c standard trace information
+// https://www.w3.org/TR/trace-context/
+const TraceparentHeaderName = "traceparent"
```

Comment:
```suggestion
// traceparentHeaderName is the HTTP header field for w3c standard trace information
// https://www.w3.org/TR/trace-context/
const traceparentHeaderName = "traceparent"
```
Lets keep this unexported unless neccessary.